example/SherpaExample/UserGuide.json

```diff
@@ -45,9 +45,21 @@
 				{
 					"key": "body",
 					"title": "Body Content",
-					"body": "The body of an article may be a string of any length, and can contain basic HTML markup (such as <code>&lt;a&gt;</code>, <code>&lt;strong&gt;</code> or <code>&lt;em&gt;</code>) which is converted for display when viewing the article.",
+                         "body": "The body of an article may be a string of any length, and can contain basic HTML markup (such as <code>&lt;a&gt;</code>, <code>&lt;strong&gt;</code> or <code>&lt;em&gt;</code>) which is converted for display when viewing the article.",
 					"related_articles": ["key", "article-title", "minimum-build", "maximum-build", "related-articles"]
 				},
+                 {
+                     "key": "html",
+                     "title": "HTML Content",
```

Comment:
I'm not certain what the purpose of adding the "HTML Content" article here is, it seems to be a minor rewrite of the "Body Content" article. If you'd like to include the image as an example, I suggest adding it there.